esda/moran.py

```diff
@@ -67,7 +67,8 @@ class Moran:
     transformation  : string
                       weights transformation,  default is row-standardized "r".
                       Other options include "B": binary,  "D":
-                      doubly-standardized,  "U": untransformed
+                      doubly-standardized,  "O": restore original transformation 
+                      (applicable only if ``w`` is  passed as ``W``)
```

Comment:
This clarification should be present in all docstrings below.

Reply:
Agreed. Sorry--I'm a bit new at the pull request thing. I updated all of the docstrings in my own repository, but those changes weren't reflected here. How do I make the changes show up in the pull request itself, without opening a new PR? Thanks for bearing with me.

Reply:
Actually, I think I've figured it out. Thanks again for bearing with me as I learn!